### PR TITLE
Add metrics exporter and configuration for SystemMetrics

### LINKED ISSUE #40: Configure MetricsExporter in otel-launcher-py
The launcher must configure metrics exporting. It should also configure the system metrics instrumentation

there’s a similar example for Go here: https://github.com/lightstep/otel-launcher-go/pull/13/files#diff-41ec8b3c433f7893f3bf0e1ffc942facR344﻿

### PR DESCRIPTION
**Pull request by ocelotl**
Fixes #40

### CODE REVIEW

**Review by codeboten — CHANGES_REQUESTED** [review]
This is a great start, would like to see some unit tests around metrics configuration, requesting for change to ensure we don't merge before the new packages are released.

File: noxfile.py [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+        "#egg=opentelemetry-exporter-otlp"
+        "&subdirectory=exporter/opentelemetry-exporter-otlp"
+    )
+    session.install(
+        "-egit+https://github.com/ocelotl/opentelemetry-python.git"
+        "@660ddb118790214e86fe93ed1e61e9eecb822715"
+        "#egg=opentelemetry-instrumentation-system-metrics"
+        "&subdirectory=instrumentation/"
+        "opentelemetry-instrumentation-system-metrics"
+    )
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply noxfile.py]: Fixed

File: src/opentelemetry/launcher/configuration.py [hunk 2]
```diff
@@ -284,6 +295,37 @@ def configure_opentelemetry(
...
+
+    if _env.str("OTEL_PYTHON_METER_PROVIDER", None) is None:
+        # FIXME now that new values can be set in the global configuration
+        # object, check for this object having a tracer_provider attribute, if
+        # not, set it to "sdk_tracer_provider" instead of using
+        # set_tracer_provider, this in order to avoid having more than one
+        # method of setting configuration.
+        set_meter_provider(MeterProvider())
+
+    if metric_exporter_insecure:
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply src/opentelemetry/launcher/configuration.py]: Fixed

**Review by codeboten — APPROVED** [review]
Only missing an update to the README, otherwise looks good

File: src/opentelemetry/launcher/configuration.py [hunk 3]
```diff
@@ -103,6 +129,7 @@ def configure_opentelemetry(
     log_level: str = _OTEL_LOG_LEVEL,
     span_exporter_insecure: bool = _OTEL_EXPORTER_OTLP_SPAN_INSECURE,
     metric_exporter_insecure: bool = (_OTEL_EXPORTER_OTLP_METRIC_INSECURE),
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply src/opentelemetry/launcher/configuration.py]: fixed

File: src/opentelemetry/launcher/configuration.py [hunk 4]
```diff
@@ -369,10 +369,13 @@ def configure_opentelemetry(
...
     )
 
-    SystemMetrics(lightstep_otlp_metrics_exporter, system_metrics_config)
+    system_metrics = SystemMetrics(
+        lightstep_otlp_metrics_exporter, system_metrics_config
+    )
 
     get_meter_provider().start_pipeline(
-        get_meter(__name__),
+        # get_meter(__name__),
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply src/opentelemetry/launcher/configuration.py]: fixed

File: src/opentelemetry/launcher/configuration.py [hunk 5]
```diff
@@ -31,28 +31,36 @@
...
-from opentelemetry.trace.propagation.tracecontext import (
-    TraceContextTextMapPropagator,
-)
 from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.system_metrics import SystemMetrics
+from opentelemetry.launcher.metrics import LightstepOTLPMetricsExporter
 from opentelemetry.launcher.tracer import LightstepOTLPSpanExporter
 from opentelemetry.launcher.version import __version__
+from opentelemetry.metrics import (
+    # get_meter,
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply src/opentelemetry/launcher/configuration.py]: fixed

File: src/opentelemetry/launcher/configuration.py [hunk 6]
```diff
@@ -31,28 +31,36 @@
...
+    TraceContextTextMapPropagator,
+)
 
 _env = Env()
 _logger = getLogger(__name__)
 
 _DEFAULT_OTEL_EXPORTER_OTLP_SPAN_ENDPOINT = "ingest.lightstep.com:443"
 _DEFAULT_OTEL_EXPORTER_OTLP_METRIC_ENDPOINT = (
-    "ingest.lightstep.com:443/metrics"
+    "ingest.lightstep.com:443"
```
[the remark this answers is not in the document]
Comment (ocelotl) [reply src/opentelemetry/launcher/configuration.py:62]: fixed

**Pull request merged**